src/views/Home/components/DiningBalance/index.js

```diff
@@ -58,6 +58,15 @@ export default class DiningBalance extends Component {
       content = <GordonLoader />;
     } else {
       if (this.facStaffBalance) {
+        //balance color will be green or yellow (low balance) or orange (really low balance)
```

Comment:
So, does this 'if' block only run if facStaffBalance is non-zero?

Reply:
Oh, your comment is slightly unclear. It sounds like a color of green or yellow represents low balance. Replacing the first 'or' with a comma would probably sound right.

Reply:
Oh, I realized a better question to ask than my first one is "does this 'if' block only run if facstaffBalance is non-null?"

Reply:
The "if" statement was pre-existing.  An string will evaluate to False in a Boolean expression if it is empty, otherwise it will be True (https://stackoverflow.com/questions/8692982/in-javascript-is-an-empty-string-always-false-as-a-boolean). 

Comments have been rewritten!